files/bash_aliases

```diff
@@ -37,9 +41,28 @@ alias apacherestart='sudo httpd -e info -k restart'
 alias memcached-start='memcached -I2m -d start'
 alias mariastart='brew services start mysql'
 alias mariastop='brew services stop mysql'
+alias apache-logs='cd /usr/local/var/log/httpd'
+alias codecept='php ~/work/testing-toolkit/vendor/codeception/codeception/codecept'
 
 ## OPEN FILE WITH TEXTWRANGLER
 function textWranglerOpen() {
 	open -a TextWrangler $1
 }
 alias edit=textWranglerOpen
+
+## Misc
+goodMorningClosure() {
+	say "Good morning motherfucker!"
+	open -a "Google Chrome" https://mail.google.com/ https://github.com
+	open -a "Slack"
+	say "Updating repositories"
```

Comment:
Can we replace `say` with `echo`? What do you think about this

Reply:
"say" is intentional since it give at least +6 awesomeness points to the whole script :D 
Np, i'll change it to echo. 👍